### PR TITLE
getting reviews for current user

### PR DESCRIPTION
**Pull request by 360devin360**
Error: ambiguous url
due to ReviewImages and SpotImages url

going to give alias to spotImage.spotId to try to get around

### CODE REVIEW

File: backend/db/models/review.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ module.exports = (sequelize, DataTypes) => {
         foreignKey:'spotId',
         onDelete:'cascade'
       })
+      Review.belongsTo(models.User,{
+        foreignKey:"userId",
+        onDelete:'cascade'
```
Comment (AnthonyBronca) [inline backend/db/models/review.js:22]: Keep in mind that cascade may cause unintended effects if used here

**Review by AnthonyBronca — COMMENTED** [review]
Just a possible bug. Look at review comments for thoughts

File: backend/db/models/review.js (hunk 1 above)
Comment (360devin360) [reply backend/db/models/review.js:22]: will check that today

**Review by AnthonyBronca — COMMENTED** [review]
Looks good

**Comment by 360devin360** [comment]
got it

**Pull request merged**